<launch>
  <arg name="manager" default="realsense_ros_camera_manager"/>

  <include file="$(find realsense_ros_camera)/launch/camera.launch">
    <arg name="manager" value="$(arg manager)"/>
    <arg name="depth_width" value="320"/>
    <arg name="depth_height" value="240"/>
    <arg name="color_width" value="320"/>
    <arg name="color_height" value="240"/>
    <arg name="enable_depth" value="true"/>
    <arg name="enable_color" value="true"/>
  </include>

  <node pkg="nodelet" type="nodelet" name="depthimage_to_laserscan" args="load depthimage_to_laserscan/DepthImageToLaserScanNodelet $(arg manager) --no-bond">
    <param name="scan_height" type="int" value="10" />
    <param name="range_min" type="double" value="0.35" />
    <param name="range_max" type="double" value="4.0" />
    <remap from="image" to="/camera/depth/image_raw"/>
    <remap from="camera_info" to="/camera/depth/camera_info"/>
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find realsense_ros_camera)/rviz/pointcloud_and_laser_view.rviz" required="true" />
</launch>